posts/leveraged-learning-with-chat-gpt.md

```diff
@@ -0,0 +1,101 @@
+
+## Why Learning is critical for Software Development
+
+Learning is a crucial part of being a Software Developer - technology has always been evolving and although there are certain stable methodologies that have stood the test of time, with 3 years of experience under my belt now I’m no stranger to the fact that there is still a lot of knowledge left to learn in my career. Furthermore since I generally specialise more towards frontend development, anyone that's familiar with it would know how fast the frameworks come and go. But now more than ever with the onset of AI looming, I believe learning is becoming even more important because if one doesn’t keep up to date with the potentially rapid changing pace of development, then certainly the risk in career could be a lot greater.
+
+## Standard Learning Process
+
+So let's start off with a typical situation of how the learning occurs the traditional way - without ChatGPT. I’m currently taking part in a structured learning initiative as part of Ippon called the Black Belt Program, that allows one to specialise in a chosen field and receive mentorship and guidance towards mastery of an area. As part of my current “Green belt” in the Full Stack frontend specialisation, I need to be able to explain several topics to my mentor - for example regarding JWT Token as well as E2E Testing.
+
+If we apply principles from popular learning frameworks such as Blooms taxonomy, we can see how they also apply to learning new technology:
+
+Back in University I minored in Psychology where I learnt more about the cognitive science behind learning. This is where I learnt about **Blooms Taxonomy, **a framework that promotes deeper understanding over the standard rote learning and memorization of facts, and also helped explain why cramming for exams by simply recalling facts would not lead to long term understanding or even be very applicable in an exam situation since it **lacked context.** So since then I’ve always tried to use this as a kind of guideline to assist me in making sure I am better aware of where I stand in the learning process for a new concept as well as how I should be looking to improve.
+
+![Blooms Taxonomy](../images/2023/05/blooms-taxonomy.png)
```

Comment:
I believe you need to pass in the full URL for these to work. It's like `raw.github` or something similar.